client/src/competition/competition-details-modal/CompetitionDetailsModal.vue

```diff
@@ -0,0 +1,131 @@
+<template>
+  <ModalWrapper>
+    <ModalContent>
+      <ModalHeader class="competition-details-modal__header">
```

Comment:
What is the reason for direct defining ```class="competition-details-modal__header"``` instead of extending from style component? because ```competition-details-modal__header``` doesn't look like some global class

Reply:
Very good point, It is an obsolete class, initially, I wrote styles differently. Removed.